parse.go

```diff
@@ -143,6 +144,10 @@ func Parse(st interface{}) error {
 
 		if err == nil {
 			defaultValue = defaultTag.Name
+
+			if len(defaultTag.Options) > 0 {
+				defaultValue = fmt.Sprintf("%s,%s", defaultValue, strings.Join(defaultTag.Options, ","))
+			}
```

Comment:
rest of elements after the first "," where considered "options" instead of part of the default value